docs/iot/guides/update-control-api.md

```diff
@@ -0,0 +1,134 @@
+# Tizen Update Control Native API
+
+- [Prerequisites](#prerequisites)
+- [Connecting to Server](#connecting-to-server)
+- [Checking the Latest Version](#checking-the-latest-version)
+- [Downloading and Updating to New Version](#downloading-and-updating-to-new-version)
+- [Making an Update Reservation](#making-an-update-reservation)
+- [Update Properties](#update-properties)
+
+You can control the system software version of IoT device using update control API.
+The control of system software version includes:
+- Checking the latest version
+- Downloading and updating to the latest version
+- Making a reservation for update
+
+The update procedure and the server might be different according to the platform developer.
+Therefore, this API just offers general functions for update, and platform developers must develop the application using update control API properly.
```

Comment:
Question. Isn't "update control" the name of the API? If so, it should be "Update Control", using capital letters.

Reply:
@mijin85cho Okay. Done.